test/common/common.go

```diff
@@ -36,6 +43,8 @@ func init() {
 	flag.StringVar(&UserNamespace, "user_namespace", "policy-test", "ns on hub to create root policy")
 	flag.StringVar(&ClusterNamespace, "cluster_namespace", "local-cluster", "cluster ns name")
 	flag.IntVar(&DefaultTimeoutSeconds, "timeout_seconds", 30, "Timeout seconds for assertion")
+	flag.BoolVar(&ManuallyPatchDecisions, "patch_decisions", true, "Whether to 'manually' patch PlacementRules with PlacementDecisions (set to false if the PlacementRule controller is running)")
```

Comment:
The `e2e` tests are currently only run against kind clusters. I want to run them against real clusters in the future.

Reply:
We run `run-e2e-tests.sh` in Travis. So I think this could be set there?
https://github.com/stolostron/governance-policy-framework/blob/11a678376f6817ce1e01c00f7295d14ede419c25/.travis.yml#L16

Reply:
`--patch_decisions` does not need to be set in those Travis stages / the run-e2e-tests.sh script, wince the default behavior is what we need there. IF we ran the `e2e-tests` target against a cluster with the placement rule controller running (eg a full ACM cluster) then we *could* set `--patch_decisions=false`.

From my initial testing on a real cluster, having the tests patch the decisions wouldn't cause any problems. And I don't see any time we run these e2e tests against a real cluster automatically.